### PR TITLE
E2e delete additional docs second and final version

### PR DESCRIPTION
**Pull request by jacobrkandc**
## Issue ticket number and link

A2-1641 TEST E2E Automation - Remove doc when not the last version
A2-1821 Test E2E - Remove doc when last version

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [x ] I have performed a self-review of my own code
- [x ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js [hunk 1]
```diff
@@ -34,6 +34,60 @@ describe('manage docs on appellant case', () => {
...
+	it('remove doc when not final version', () => {
+		cy.createCase().then((caseRef) => {
+			happyPathHelper.uploadDocAppellantCase(caseRef);
+			caseDetailsPage.clickAddAdditionalDocs();
+			caseDetailsPage.uploadSampleFile(sampleFiles.document);
+			caseDetailsPage.checkFileNameDisplays('sample-file.doc');
+			caseDetailsPage.clickButtonByText('Continue');
+			caseDetailsPage.clickButtonByText('Confirm');
+			caseDetailsPage.clickButtonByText('Confirm');
+			caseDetailsPage.checkAdditonalDocsAppellantCase('sample-file.doc');
```
Comment (RHARV018) [inline appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js:67]: For all of the lines which reference 'sample-file.doc' you could use our new method - (sampleFiles.doc) - add in the variable at the top of the spec file before the first 'it'. variable is let sampleFiles = caseDetailsPage.sampleFiles;

File: appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js [hunk 2]
```diff
@@ -34,6 +34,60 @@ describe('manage docs on appellant case', () => {
 		});
 	});
 
+	it('upload doc and remove when final version', () => {
+		cy.createCase().then((caseRef) => {
+			happyPathHelper.uploadDocAppellantCase(caseRef);
+			caseDetailsPage.clickAddAdditionalDocs();
+			caseDetailsPage.uploadSampleFile(sampleFiles.document);
+			caseDetailsPage.checkFileNameDisplays('sample-file.doc');
```
Comment (PeterBridgerKC) [inline appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js:42]: Same point as Rachel made below

File: appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js (hunk 1 above)
Comment (jacobrkandc) [reply appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js]: I've updated the relevat method to use the variable (sampleFiles.document)

File: appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js (hunk 2 above)
Comment (jacobrkandc) [reply appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js]: Updated to reflect Rachel's comments above

File: appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js [hunk 3]
```diff
@@ -34,6 +34,60 @@ describe('manage docs on appellant case', () => {
...
+			caseDetailsPage.checkAdditonalDocsAppellantCase('No documents available');
+		});
+	});
+
+	it('remove doc when not final version', () => {
+		cy.createCase().then((caseRef) => {
+			happyPathHelper.uploadDocAppellantCase(caseRef);
+			caseDetailsPage.clickAddAdditionalDocs();
+			caseDetailsPage.uploadSampleFile(sampleFiles.document);
+			caseDetailsPage.checkFileNameDisplays('sample-file.doc');
```
Comment (RHARV018) [inline appeals/e2e/cypress/e2e/back-office-appeals/manageDocsAppellantCase.spec.js:63]: sorry to be pedantic, but you can use sampleFiles.document for line 63 too

File: appeals/e2e/cypress/e2e/back-office-appeals/withdrawAppeal.spec.js [hunk 4]
```diff
@@ -36,7 +36,7 @@ describe('Withdraw a Has appeal', () => {
 			caseDetailsPage.selectRadioButtonByValue('Redacted');
 			caseDetailsPage.clickButtonByText('Continue');
 			caseDetailsPage.checkAnswerWithdrawalRequest('Withdrawal request', 'sample-file.doc');
```
Comment (RHARV018) [inline appeals/e2e/cypress/e2e/back-office-appeals/withdrawAppeal.spec.js:38]: same for line 38

**Review by RHARV018 — DISMISSED** [review]
Looks good to me!

**Review by PeterBridgerKC — DISMISSED** [review]

**Pull request closed without merging**